### PR TITLE
Separate readme and logs

### PR DESCRIPTION
**Pull request by CFGrote**
This PR restructures the content of the top level readme and performs various cleanups

- Convert readmes to org and transclude from run logs.
- Remove readme in Run 1 logdir.
- Replace .md by .org, update all plots, move all log to one logbook.
- Remove more mds and document generation of reduced size graphs.
- Update logbook
- Update readme, logbook
- Update log

### CODE REVIEW

**Comment by CFGrote** [comment]
Hey @marimeireles , can you give this a quick read and approval before merging? Thanks :wave:

File: queries/20241210T091800_analyze_timings.ipynb
Comment (marimeireles) [inline queries/20241210T091800_analyze_timings.ipynb:1]: I've never done this before so I'm really not sure how to review `ipynb`s. Not sure if there's a good way?
In `20241210T091800_analyze_timings`, at this cell:
```
ax.set_xlabel('')
ax.set_ylabel('time')
ax.set_title("Query "+DATASET.replace("_", " ").replace("-", ": "))ax
```
`ax` is being used before it's defined, I think it's a matter of rearranging cells.
There's also an empty cell in this notebook.

File: Logbook.org [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+:END:
+- Servers: ontop, fuseki
+- number of query repeats: 30
+- client ip: 128.176.254.12
+- client hostname: micropop046
+- client software: apache-jena-5.2.0/bin/rsparql
+- rsparql version reported as 4.2.0
+- connection: vpn to uni-muenster
+- note: all queries were run once about 30min before. Noting reduced
+  query time on the 00-construct and other queries. Caching?
```
Comment (marimeireles) [inline Logbook.org]: We can add there was no available API for controlling caching within servers so the best we could was hot cache?

File: Logbook.org [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+:PROPERTIES:
+:ID:       9fe63540-6427-44bc-8e10-07d53b0e0805
+:END:
+In all queries, ontop's query response time is faster than fuseki. Must be aware of caching, hence should try other query submission strategy (-> see Run 2, Run 3)
+
+** Run 2: Loop over 10 queries, 2 servers, 10 rounds in that order
+:PROPERTIES:
+:ID:       825c8799-bb59-4b89-aa22-1b4b2d1279c8
+:END:
+By changing the order with respect to Run1, we try to eliminate the effect of caching, as each server never sees the same two queries immediately one after the other, wherea
```
Comment (marimeireles) [inline Logbook.org:78]: grammar:
wherea -> whereas
and erase line break

File: Logbook.org [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+#+end_example
+
+*** Results
+     :PROPERTIES:
+     :CUSTOM_ID: results-1
+     :ID:       e49db261-4cf9-4b44-823c-6045b3984f17
+     :END:
+Timings come out differently: for many queries, now fuseki is faster,
+ontop factor 2 slower.
+[[file:queries/saved_runs/20241210T113800/facet_walltime.png]]
```
Comment (marimeireles) [inline Logbook.org:141]: Any idea why?

**Review by marimeireles — COMMENTED** [review]
Just minor comments.

File: queries/20241210T091800_analyze_timings.ipynb
Comment (CFGrote) [reply queries/20241210T091800_analyze_timings.ipynb:1]: i cleaned up the notebooks and removed unneeded cells.

File: Logbook.org (hunk 2 above)
Comment (CFGrote) [reply Logbook.org:78]: done in 32df07f

File: Logbook.org (hunk 1 above)
Comment (CFGrote) [reply Logbook.org]: done in 32df07f

File: Logbook.org (hunk 3 above)
Comment (CFGrote) [reply Logbook.org:141]: Well this was what we expected, right? ontop has the additional task to rewrite a received sparql query into sql and re-translate the sql response into rdf.

Comment (CFGrote) [reply Logbook.org:141]: added short discussion to Run 2 section in commit 906cb3e

**Pull request merged**